.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  # Ruby dependencies
+  - package-ecosystem: "bundler"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 10
+  # JavaScript dependencies  
+  - package-ecosystem: "npm"
```

Comment:
we should use "npm" for pnpm as well: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#package-ecosystem-